R/meanimpute.R

```diff
@@ -1,6 +1,18 @@
-#' Meanimputation
+#' Meanimpute
+#'
+#' Replace NA's with mean value
+#' @param x a numeric vector
+#' @return new vector, where NA's are replace by mean \code{x}
+#' @examples
+#' example_vector=c(1,5,NA,NA)
+#' meanimpute(example_vector)
 #' @export
 meanimpute <- function(x) {
+  
+  if(is.null(x)) {stop("Input vector cannot be NULL.")}
+  if(all(is.na(x))) {stop("Input vector should contain at least one numeric element.")}
```

Comment:
Being a bit picky here: ´Input vector should contain at least one numeric element which is not NA´.